### DISCUSSION TITLE
89

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#41] [General]
# 潮流周刊第89期 - 去避暑吧

 照片摄于周末去杭州周边避暑的一个溶洞，外面超级热，坐船两分钟进去就彻底凉快了，这张照片有些感觉。

https://weekly.tw93.fun/posts/89-%E5%8E%BB%E9%81%BF%E6%9A%91%E5%90%A7

**Comment by yzh990918** [comment]
想问下作者录GIF的软件是什么
